airsonic-main/src/main/java/org/airsonic/player/service/PodcastService.java

```diff
@@ -662,6 +684,12 @@ private boolean isEpisodeDeleted(PodcastEpisode episode) {
         return episode == null || episode.getStatus() == PodcastStatus.DELETED;
     }
 
+    // include episode == null to avoid attempts to download episodes for deleted channels
+    private boolean isEpisodeQueued(PodcastEpisode episode) {
```

Comment:
rename to skipDownload?

also add PodcastStatus.SKIPPED?

Reply:
I've thought of several names, none of which really cover the true intention of this test which is both to avoid double downloads as well as to keep it from downloading deleted podcasts. `skipDownload` doesn't really get across the intention, it reeks of `PodcastStatus.SKIPPED` with which is has nothing to do - that is meant for episodes which have never been selected for downloading. Since episodes which are never selected for download do not reach this function `PodcastStatus.SKIPPED` does not need to be part of this test.